Fix apptesting enablement (#8905)
firebase/firebase-tools: CHANGELOG.md

- Fixed ext:export command so that it correctly returns system params in the .env file (#8881)
- Fixed an issue where the MCP server could not successfully use Application Default Credentials. (#8896)

> - Fixed an issue where we enabled the incorrect API for apptesting usage.

Reviewer: ```suggestion
- Fixed an issue where the incorrect API was enabled  for `apptesting` commands.
```

> - Fixed an issue where we enabled the incorrect API for apptesting usage.

Reviewer: ```suggestion
- Fixed an issue where the incorrect API was enabled  for `apptesting` commands.
```
